synth/src/datasource/relational_datasource.rs

```diff
@@ -69,20 +69,22 @@ pub trait RelationalDataSource: DataSource {
         for column_info in column_infos {
             if let Some(value) = first_valueset.get(&column_info.column_name) {
                 match (value, &*column_info.data_type) {
-                    (
-                        Value::Number(Number::U64(_)),
-                        "int2" | "int4" | "int8" | "int" | "integer" | "smallint" | "bigint",
-                    ) => warn!(
+                    (Value::Number(Number::U64(_)), "int2" | "int4" | "int8") => warn!(
```

Comment:
I don't know why the `| "int" | "integer" | "smallint" | "bigint"` was needed since Postgres does not use them. Maybe I should just create a warnings test for Mysql to be on the safe side...